.gitignore

```diff
@@ -165,3 +165,4 @@ cython_debug/
 /.pants.workdir.file_lock*
 
 # End of custom ignore
+credentials.json
```

Comment:
This should be before "# End of custom ignore"... I'm assuming it's a custom ignore